packages/outputarea/src/widget.ts

```diff
@@ -386,7 +386,10 @@ export class OutputArea extends Widget {
       }
       output.renderModel(model).catch(error => {
         // Manually append error message to output
-        output.node.innerHTML = `<pre>Javascript Error: ${error.message}</pre>`;
+        const pre = document.createElement('p');
```

Comment:
Is this change from `<pre>` to `<p>` intended?

Reply:
No, thanks for catching this.